### DISCUSSION TITLE
Poetry keeps installing Windows requirements on Macs

### DISCUSSION THREAD

**Discussion by tyteen4a03** [python-poetry/poetry#6957] [Q&A] [answered]
Hi, I have 2 wheels, `wheelme-1.11.0-cp310-cp310-macosx_10_14_x86_64.whl` and `wheelme-1.11.0-cp38-cp38-win_amd64.whl`, both hosted on my private Nexus3 instance.

For some reason when I execute `poetry add --source nexus3 wheelme` it insists on downloading the Windows wheel even though I'm on macOS (M1, but running x64 Python). How do I force Mac Poetry to download the Mac wheel and Windows Poetry the Windows wheel?

**Comment by neersighted** [comment, marked as answer]
Are you sure you don't mean `poetry add` instead of `poetry install`? `add` will download wheels for all platforms as we need the metadata (requirements, file for hashing) when locking.

**Reply by tyteen4a03** [reply]
Yes sorry I do mean `add` - the windows download scared me. 

Metadata is now updated :)
